### PR TITLE
Test also using chrome

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

**Comment by coolo** [comment]
the strange failure in 18-test-details might be the killer for phantomjs :(

**Comment by Martchus** [comment]
I can not reproduce the the failure in 18-test-details with phantomjs (maybe sporadic issue).

BTW:

```
time SELENIUM_CHROME=0 prove -v t/ui/18-tests-details.t
real    0m9,985s
user    0m5,492s
sys     0m0,340s
```

```
time SELENIUM_CHROME=1 prove -v t/ui/18-tests-details.t
real    0m13,696s
user    0m5,360s
sys     0m0,228s
```


So executing tests in non-headless Chromium takes only a few seconds longer. But when accumulating this for all tests it might be almost a minute. Startup-phase where URL is just `data:,` takes actually quite long.

**Comment by coolo** [comment]
I can't reproduce this either - but it's 100% happening on 14.04 ;(

**Comment by coolo** [comment]
AssetPack 1.36 is in the making with a fix

**Comment by coolo** [comment]
For now the additional chrome tests give me the additional coverage I'm expecting in full-stack. But my goal is to get back to only pnantomjs with the same coverage. But for this I need to find the actual problem seen.

File: t/ui/01-list.t [hunk 1]
```diff
@@ -106,61 +96,52 @@ is_deeply(\@header, \@expected, 'test report can be adjusted with query paramete
...
 is(@links, 2, 'only two links (icon, name, no restart)');
 
 # Test 99926 is displayed
-is($driver->find_element('#results #job_99926 .test .status.result_incomplete', 'css')->get_text(),
-    '', '99926 incomplete');
+is($driver->find_element('#results #job_99926 .test .status.result_incomplete')->get_text(), '', '99926 incomplete');
 
 # parent-child
-my $child_e = $driver->find_element('#results #job_99938 .parent_child', 'css');
+my $child_e = $driver->find_element('#results #job_99938 .parent_child');
```
Comment (aaannz) [inline t/ui/01-list.t]: Does this work? find_element defaults to xpath.

Comment (aaannz) [inline t/ui/01-list.t]: Ah, I see. Changed default_finder.

**Pull request merged**